### DISCUSSION TITLE
Practica1 Front End - Requerimientos. (avance)

### DISCUSSION THREAD

**Discussion by Eholguin05** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#36] [Show and tell]
Que tal, agradezco de antemano su tiempo, aclaro que aun no termino y lo que dejo acontinuacion es un draft esperando que le sirva a alguien mas y sin ningun deseo de confundir a nadie, si lo hago háganmelo saber y me disculpo de antemano si sucede.

Acepto sus criticas, retroalimentación, etc. etc...
Este se encuentra en mi reporsitorio, si gustan hacemos un meeting y hacemos la practica en grupo. 
[Follow me]
Fork as needed.
[Requerimientos.doc](https://github.com/Eholguin05/Practica1FrontE-Requeremientos/blob/9b8460d793bd712c9c31906bcd48202d7f832c28/Requerimientos.doc)

**Comment by Eholguin05** [comment]
(TIENEN HASTA EL 25/02/22 PARA SUBIR SUS EVIDENCIAS)...
Jueves a las 11:pm nos juntamos para ver como quedaron nuestras practicas y subirlas. Los que gusten pásenme su correo para mandarles un meeting request por team.

**Reply by YomairaBarrera** [reply]
Hola compañero si te refieres a una llamada por el Teams de Microsoft, no esta permitido te pueden sacar del curso si lo haces, te recomiendo que cheques los acuerdos de convivencia que nos dejaron en el onboarding para que no vayas a tener problemas.

**Reply by Eholguin05** [reply]
Oh tienes razon, no recordaba. muchas gracias!

**Reply by FerminMtz** [reply]
Hola, gracias por tu aportación, una duda que herramienta utilizaste para el diagrama de actividades, saludos.

**Reply by lunasaurio** [reply]
Compañeros, yo tengo una duda, cual es la práctica? De que va? Estoy un poco confundido con eso. Saludos
